docs/templates/nrelatb_child.rst.jinja

```diff
@@ -0,0 +1,78 @@
+{% extends "data_source_parent.rst.jinja" %}
+
+{% block background %}
+NREL publishes `Annual Technology Baseline (ATB) <https://atb.nrel.gov>`__ data for the
+`Electricity <https://atb.nrel.gov/electricity>`__ and
+`Transportation <https://atb.nrel.gov/transportation>`__ sectors.
+
+The NREL Annual Technology Baseline (ATB) provides a consistent set of technology cost
+and performance data for energy analysis.
+
+To inform electric and transportation sector analysis in the United States, each year
+NREL provides a robust set of modeling input assumptions for energy technologies (the
+Annual Technology Baseline) and a diverse set of potential electricity generation
+futures or modeling scenarios (Standard Scenarios).
```

Comment:
I know NREL mentions Standard Scenarios in the same breath as ATB but it seems weird for us to list it when we haven't integrated it
```suggestion
Annual Technology Baseline).
```